docs/_data/portland-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
+
+talks_day1:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
+    title: Write the Docs Team - Introduction to Write the Docs
+  - time: '9:45'
+    title: "<b>Unconference Starts</b>"
+  - time: '9:45'
+    slug: writing-a-perfect-technical-tutorial-jessica-garson
+  - time: '10:30'
+    title: "Break"
+  - time: '10:45'
+    slug: always-complete-never-finished-daniele-procida
+  - time: '11:30'
+    slug: a-guide-to-getting-started-in-open-source-abigail-mccarthy
+  - time: '12:15'
+    title: Lunch (60 mins)
+  - time: '1:15'
+    title: Lightning Talks (30 mins)
+  - time: '2:00'
+    slug: beyond-metrics-using-maturity-models-to-develop-a-docs-strategy-sarah-r-rodlund
+  - time: '2:45'
+    slug: shuffle-ball-change-sashay-your-way-to-clearer-api-documentation-rachael-stavchansky
+  - time: '3:30'
+    title: "Break"
+  - time: '3:45'
+    slug: putting-the-tech-in-technical-writer-swapnil-ogale
+  - time: '4:30'
+    slug: the-secret-history-of-libraries-paris-buttfield-addison
+  - time: '5:15'
```

Comment:
Do we want to have the wrap-up prerecorded and inserted before the final Q&A? How would we indicate this?

Reply:
This is a good question, not sure the best way to handle it in the schedule.

Reply:
I can't remember why we wanted to do this?

Reply:
Because otherwise we have to herd people back to the main stage for the closing which feels weird

Reply:
Yeah. If they haven't got it by the end of the day...

But otherwise, we could add "& QA" to *each* talk, and add a specific "Q&A for X talk" line after?

Reply:
To be honest, after thinking this through, I think it is a non issue. Folks are gonna watch what they want either way. 

* Structure like any other talk + Q&A, final Outro back on the main stage (link will be there so folks can watch if they miss it?)
* Add a single 5 min Q&A slot after the wrap up (on the cal) and add a mention of it to the MC intro for the final talk

Either seems fine to me.

Reply:
I think we don't need to decide now. People aren't watching the schedule so closely that they'll notice any changes like this around the closing. Let's ship it 👍